apps/builder/app/builder/features/settings-panel/expandable-text-field.tsx

```diff
@@ -0,0 +1,100 @@
+import { forwardRef, useState, type ComponentProps } from "react";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogTitle,
+  DialogTrigger,
+  Flex,
+  Grid,
+  SmallIconButton,
+  TextArea,
+  rawTheme,
+  theme,
+} from "@webstudio-is/design-system";
+import { CrossIcon, MaximizeIcon, MinimizeIcon } from "@webstudio-is/icons";
+
+export const ExpandableTextField = forwardRef<
```

Comment:
we clearly should be able to reuse CodeEditorDialog here to maintain expanded code editor in a single place

Reply:
also just commited this, it was forgotten to remove, just in case you want to cherry pick it into your pr https://github.com/webstudio-is/webstudio/pull/2949/commits/73927a48f7d96929028ef40301c86f6e1b3f29c5

Reply:
It's really much easier to keep them separate than reuse

Reply:
maybe it seems that way, but I have many more features planned for this expanded view and I 100% believe this is going to be unmaintainable

Reply:
it's just a regular dialog with specified width and height. I'm guessing your plans related to code editing so even less reasons to unify it.

Reply:
🤷